guacamole/src/main/webapp/translations/fr.json

```diff
@@ -130,8 +132,15 @@
 
     },
 
+    "DATA_SOURCE_DEFAULT" : {
+        "NAME" : "Standard (XML)"
+    },
+
     "FORM" : {
 
+        "FIELD_PLACEHOLDER_DATE" : "DD-MM-YYYY",
```

Comment:
This still needs to be "YYYY-MM-DD", or the French-language equivalent. This is not a date format string which affects display, but rather the format that the Guacamole webapp expects the user to conform to. If French uses different abbreviations for this pattern, then that is what would be translated here, but the placeholder still needs to guide the user to enter the year, month, and day, in that order, separated by dashes.